src/components/Chat/ChatActions.js

```diff
@@ -0,0 +1,21 @@
+import { types } from './ChatActionTypes';
+import { sendMessage } from "../../services/socket";
+
+export const resetChat = () => ({
+    type: types.RESET_CHAT
+});
+
+export const sendChatFailure = error => ({
+    type: types.SEND_CHAT_FAILURE,
+    error
+});
+
+export const addMessage = (username, message) => ({
+    type: types.ADD_MESSAGE,
+    username,
+    message
```

Comment:
I think it would also look nicer if you made the object beforehand e.g. `{ username, message }` rather than passing in the attributes and creating the object in the reducer